README.md

```diff
@@ -10,13 +10,17 @@ database versions, including Oracle.
 
 ## Quickstart
 
-Clone this repository somewhere. Also make sure you have docker and docker-compose installed.
+Clone this repository and ensure you have docker and docker-compose installed. If
+you are using Windows, see the [notes below](#usage-on-windows) before continuing.
 
-Ensure that the `DJANGO_PATH` variable points to the root of the Django repo:
+### Environment
+
+You must set the `DJANGO_PATH` environment variable to the path of your local Django checkout.
+This can either be added to the `.env` file, or interactively set in your shell, like so:
 
 `export DJANGO_PATH=~/projects/django/`
 
-If you see a docker-compose warning about it not being defined followed by an error ensure that is defined in the shell you are using.
```

Comment:
I removed this because I tested the error and it is extremely specific and could not be confused - not necessary to have here.